firstvoices/backend/tasks/alphabet_tasks.py

```diff
@@ -48,3 +53,58 @@ def recalculate_custom_order_preview(site_slug: str):
     preview["updated_entries"] = updated_entries
 
     return preview
+
+
+@shared_task
+def recalculate_custom_order(site_slug: str):
+    """
+    Returns the same format as recalculate_custom_order_preview, but actually updates the custom order and
+    title of entries in a site's dictionary by saving the entries.
+    """
+
+    site = Site.objects.get(slug=site_slug)
+    alphabet = Alphabet.objects.get(site=site)
+    results = {}
+
+    updated_entries = []
+    unknown_character_count = {}
+
+    # Return the results of the recalculation i.e. the changes in custom order and title for every entry
+    for entry in DictionaryEntry.objects.filter(site=site):
+        result = {
+            "title": entry.title,
+            "cleaned_title": "",
+            "is_title_updated": False,
+            "previous_custom_order": entry.custom_order,
+            "new_custom_order": "",
+        }
+
+        # Save the entry to recalculate custom order and clean title
+        entry.save()
+
+        cleaned_title = entry.title
+        new_order = entry.custom_order
+
+        if result["previous_custom_order"] != new_order:
+            result["new_custom_order"] = new_order
+        if result["title"] != cleaned_title:
+            result["cleaned_title"] = cleaned_title
+            result["is_title_updated"] = True
+
+        # Count unknown characters remaining in each entry, first split by character, then apply custom order
+        if "⚑" in new_order:
+            chars = alphabet.get_character_list(entry.title)
```

Comment:
You could pull lines 96-98 into a little function to get the split custom order chars for a word, right? Might help with that code smell, and it would be easier to read.